blocks/cards/cards.js

```diff
@@ -43,7 +43,7 @@ export default async function decorate(block) {
       <div class="main-text-wrapper">
         <div class="section">${mainCard.author}</div>
         ${!gdPlusCards ? '' : gdPlusIcon}
-        <div class="headline"><spam role="heading">${mainCard.title}</span></div>
+        <div class="headline"><span aria-level="3" role="heading">${mainCard.title}</span></div>
```

Comment:
@DriftingSands  Can you use h3 instead ?

Reply:
will do